### ISSUE TITLE
Add !gvideo trigger

### ISSUE THREAD

**Issue by adamstirtan** [adamstirtan/ircbot#3]
URL: https://ajax.googleapis.com/ajax/services/search/video?v=1.0&q=Paris%20Hilton

Response:

{"responseData": {"results":[{"GsearchResultClass":"GvideoSearch","title":"Paris Hilton - Good Time (Explicit) ft. Lil Wayne - YouTube","titleNoFormatting":"Paris Hilton - Good Time (Explicit) ft. Lil Wayne - YouTube","published":"Tue, 08 Oct 2013 08:40:00 -0700","content":"Music video by Paris Hilton performing Good Time (Explicit). © Cash Money  Records Inc ...","publisher":"youtube.com","tbUrl":"https://img.youtube.com/vi/K1JMjwJG2UM/default.jpg?h\u003d90\u0026w\u003d120\u0026sigh\u003d__oTsdIj6zhq-5bKVBkGtDn8bVdiQ\u003d","tbWidth":"120","tbHeight":"90","videoType":"YouTube","url":"http://www.youtube.com/watch?v\u003dK1JMjwJG2UM","playUrl":"http://youtube.googleapis.com/v/K1JMjwJG2UM\u0026source\u003duds\u0026autoplay\u003d1","duration":"227"},{"GsearchResultClass":"GvideoSearch","title":"Paris Hilton - Come Alive - YouTube","titleNoFormatting":"Paris Hilton - Come Alive - YouTube","published":"Tue, 15 Jul 2014 10:01:00 -0700","content":"Music video by Paris Hilton performing Come Alive. ... As much as I love Paris  Hilton, I feel ...","publisher":"youtube.com","tbUrl":"https://img.youtube.com/vi/6KXwYjG8txo/default.jpg?h\u003d90\u0026w\u003d120\u0026sigh\u003d__iDOQxWpZ_SeaQKgNqhLLW3lZGC0\u003d","tbWidth":"120","tbHeight":"90","videoType":"YouTube","url":"http://www.youtube.com/watch?v\u003d6KXwYjG8txo","playUrl":"http://youtube.googleapis.com/v/6KXwYjG8txo\u0026source\u003duds\u0026autoplay\u003d1","duration":"254"},{"GsearchResultClass":"GvideoSearch","title":"Paris Hilton - High Off My Love ft. Birdman - YouTube","titleNoFormatting":"Paris Hilton - High Off My Love ft. Birdman - YouTube","published":"Thu, 14 May 2015 21:01:00 -0700","content":"Music video by Paris Hilton performing High Off My Love. © 2015 Cash .... Why  do people ...","publisher":"youtube.com","tbUrl":"https://img.youtube.com/vi/rGnKkgzVpCE/default.jpg?h\u003d90\u0026w\u003d120\u0026sigh\u003d__gbrswAcJnGuazO6x4wccSMdHI2s\u003d","tbWidth":"120","tbHeight":"90","videoType":"YouTube","url":"http://www.youtube.com/watch?v\u003drGnKkgzVpCE","playUrl":"http://youtube.googleapis.com/v/rGnKkgzVpCE\u0026source\u003duds\u0026autoplay\u003d1","duration":"242"},{"GsearchResultClass":"GvideoSearch","title":"Paris Hilton talks how she \u0026#39;inspired\u0026#39; Kim ...","titleNoFormatting":"Paris Hilton talks how she \u0026#39;inspired\u0026#39; Kim ...","published":"Thu, 02 Apr 2015 10:09:22 -0700","content":"\u0026#39;I\u0026#39;m really proud!\u0026#39; Paris Hilton talks how she \u0026#39;inspired\u0026#39; Kim Kardashian\u0026#39;s meteoric  rise ...","publisher":"youtube.com","tbUrl":"https://img.youtube.com/vi/p-nkfl0ismI/default.jpg?h\u003d90\u0026w\u003d120\u0026sigh\u003d__MqmB-US7qR5Hi8bxF356hZ_qp_o\u003d","tbWidth":"120","tbHeight":"90","videoType":"YouTube","url":"http://www.youtube.com/watch?v\u003dp-nkfl0ismI","playUrl":"http://youtube.googleapis.com/v/p-nkfl0ismI\u0026source\u003duds\u0026autoplay\u003d1","duration":"354"}],"cursor":{"pages":[{"start":"0","label":1},{"start":"4","label":2},{"start":"8","label":3},{"start":"12","label":4},{"start":"16","label":5},{"start":"20","label":6},{"start":"24","label":7},{"start":"28","label":8}],"estimatedResultCount":"222000","currentPageIndex":0,"moreResultsUrl":"http://www.google.com/search?tbs\u003dvid:1\u0026source\u003duds\u0026type\u003dsearch\u0026q\u003dParis+Hilton\u0026hl\u003den\u0026start\u003d0"}}, "responseDetails": null, "responseStatus": 200}
